static/css/styles.css

```diff
@@ -252,4 +252,103 @@ button:hover {
     align-items: flex-start;
     align-content: flex-end;
     justify-content: center;
-}
\ No newline at end of file
+}
+
+.background {
```

Comment:
The code from line 257 to the end can be deleted because it duplicates  the code from line 88 to 183

Reply:
Removed duplicate styles.